recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
     - patches/0006-popcnt_fix.patch                     # [blas_impl == "mkl" and win]
 
 build:
-  number: 0
+  number: 1
```

Comment:
We haven't version 1.21.5 yet so the build number should be `0`

Reply:
I believe we already have v1.21.5 on **win-64/32** & **linux-aarch64**:

```
conda search numpy=1.21.5 --subdir linux-aarch64
Loading channels: done
# Name                       Version           Build  Channel             
numpy                         1.21.5 py310h983bb6b_0  pkgs/main           
numpy                         1.21.5  py37hd490b01_0  pkgs/main           
numpy                         1.21.5  py38hd490b01_0  pkgs/main           
numpy                         1.21.5  py39hd490b01_0  pkgs/main 
```

Reply:
Yeah, thanks! And atrifacts on win32 are built twice with build number 0:
https://repo.anaconda.com/pkgs/main/win-32
numpy-1.21.5-py310h54a06cd_0.tar.bz2
numpy-1.21.5-py310h75fcd9c_0.tar.bz2
numpy-1.21.5-py37h73ede0f_0.tar.bz2
numpy-1.21.5-py37hf27c404_0.tar.bz2
numpy-1.21.5-py38h73ede0f_0.tar.bz2
numpy-1.21.5-py38hf27c404_0.tar.bz2
numpy-1.21.5-py39h73ede0f_0.tar.bz2
numpy-1.21.5-py39hf27c404_0.tar.bz2